anago

```diff
@@ -241,7 +241,7 @@ check_prerequisites () {
 
   logecho -n "Checking cloud project state: "
   GCLOUD_PROJECT=$($GCLOUD config list project 2>/dev/null |\
```

Comment:
a less fragile way to do this would be `gcloud config get-value project 2>/dev/null`.